components/gitpod-cli/cmd/timeout-set.go

```diff
@@ -22,9 +22,9 @@ var setTimeoutCmd = &cobra.Command{
 	Short: "Set timeout of current workspace",
 	Long: `Set timeout of current workspace.
 
-Duration must be in the format of <n>m (minutes), <n>h (hours), or <n>d (days).
-For example, 30m, 1h, 2d, etc.`,
-	Example: `gitpod timeout set 1h`,
+Duration must be in the format of <n>m (minutes).
+For example, 30m`,
+	Example: `gitpod timeout set 30m`,
```

Comment:
**Suggestion(blocking):** We should remove days helper text, for sure, as it's not currently valid. But hour format is currently valid, so let's leave it here? Or, we should also update the implementation (I prefer the former).